Makefile

```diff
@@ -94,7 +94,7 @@ floo-clean:
 ###################
 
 PD_REMOTE ?= git@iis-git.ee.ethz.ch:picobello/picobello-pd.git
-PD_COMMIT ?= e1e618a872a631567dcc963eb3c987422eef4dbc
+PD_COMMIT ?= 1f60bc7569ec29e05ba996f7d7664476ac6771e2
```

Comment:
Can/should we merge this PD branch as well?

Reply:
Yes, I opened a MR in the internal repo, will update the hash here if needed.

Reply:
perfect, it is merged now. The hash is the same :)